README.md

```diff
@@ -1,2 +1,325 @@
-# swift-sdk
-High level SDK for Swift
+# Paylike's Swift SDK
+
```

Comment:
Should we add a CI badge here?
Like in the other swift packages :)

Reply:
I did initially, but as it was pointing at the other project, it was broken :D